acquia_lift.admin.unibar.inc

```diff
@@ -594,33 +594,17 @@ function acquia_lift_option_set_add_modal_callback($ajax) {
   ctools_include('modal');
   ctools_modal_add_js();
 
-  $path = drupal_get_path('module', 'acquia_lift');
+  module_load_include('inc', 'acquia_lift', 'acquia_lift.ui');
+  $types = acquia_lift_option_set_types_ui();
+  // Add a few fields specific to this display.
+  $types['personalize-element']['path'] = 'admin/structure/personalize/variations/personalize-elements/add';
+  $types['personalize-element']['modal'] = FALSE;
+  $types['personalize-block']['path'] = 'admin/structure/personalize/variations/personalize-blocks/add';
+  $types['personalize-block']['modal'] = FALSE;
```

Comment:
The keys of the array returned by the acquia_lift_option_set_types_ui() function are just 'element' and 'block'. Seems like the path at least could be included in what's returned by the function and then for the modal property you could just loop through and set them all to FALSE?